extension/js/common/ui/key-import-ui.ts

```diff
@@ -70,17 +72,47 @@ export class KeyImportUi {
     return;
   };
 
-  public static addAliasForSubmission = (email: string, submitKeyForAddrs: string[]) => {
-    submitKeyForAddrs.push(email);
+  // by unselecting, we allow to click on "Load from a file" and trigger the fineuploader again
+  public static allowReselect = () => {
+    $('input[type=radio][name=source]').prop('checked', false);
   };
 
-  public static removeAliasFromSubmission = (email: string, submitKeyForAddrs: string[]) => {
-    submitKeyForAddrs.splice(submitKeyForAddrs.indexOf(email), 1);
+  public renderEmailAliasView = async (acctEmail: string) => {
+    await Settings.refreshSendAs(acctEmail);
+    const storage = await AcctStore.get(acctEmail, ['email_provider']);
+    if (storage.email_provider === 'gmail') {
+      const { sendAs } = await AcctStore.get(acctEmail, ['sendAs']);
+      const addresses = this.filterAddressesForSubmittingKeys(Object.keys(sendAs ?? {}));
+
+      const emailAliases = Value.arr.withoutVal(addresses, acctEmail);
+      for (const e of emailAliases) {
+        for (const option of ['generate_private_key', 'submit_pubkey']) {
+          $(`.${option}_addresses`).append(
+            `<label><input type="checkbox" class="input_email_alias_${option}" data-email="${Xss.escape(e)}" data-name="${sendAs?.[e].name ?? ''}" data-test="input-email-alias-${option}-${e.replace(
+              /[^a-z0-9]+/g,
+              ''
+            )}" />${Xss.escape(e)}</label><br/>`
+          ); // xss-escaped
```

Comment:
let's move `data-test` property value into separate variable, as currently it's quite difficult to read this code:
```js
const dataTestValue = `input-email-alias-${option}-${e.replace(/[^a-z0-9]+/g, '')}`;
$(`.${option}_addresses`).append(
            `<label><input type="checkbox" class="input_email_alias_${option}" data-email="${Xss.escape(e)}" data-name="${sendAs?.[e].name ?? ''}" data-test="${dataTestValue}" />${Xss.escape(e)}</label><br/>`
          ); // xss-escaped
```